src/minio/api_notification.rs

```diff
@@ -110,3 +115,63 @@ pub struct NotificationInfo {
     pub records: Vec<NotificationEvent>,
     pub err: Option<String>,
 }
+
+impl Client {
+    /// listen_bucket_notificaion - Get bucket notifications for the bucket_name.
+    pub fn listen_bucket_notificaion(
```

Comment:
typo: listen_bucket_notification

```suggestion
    pub fn listen_bucket_notification(
```